src/avram/model.cr

```diff
@@ -247,13 +247,8 @@ abstract class Avram::Model
   macro setup_getters(columns, *args, **named_args)
     {% for column in columns %}
       {% db_type = column[:type].is_a?(Generic) ? column[:type].type_vars.first : column[:type] %}
-      def {{column[:name]}} : {% if column[:nilable] %}::Union({{db_type}}, ::Nil){% else %}{{column[:type]}}{% end %}
-        %from_db = {{ db_type }}::Lucky.from_db!(@{{column[:name]}})
-        {% if column[:nilable] %}
-          %from_db.as?({{db_type}})
-        {% else %}
-          %from_db.as({{column[:type]}})
-        {% end %}
+      def {{column[:name]}} : {{column[:type]}}{{(column[:nilable] ? "?" : "").id}}
+        {{ db_type }}::Lucky.from_db!(@{{column[:name]}})
```

Comment:
I just realized not all types have a `from_db!` method. [bool](https://github.com/luckyframework/avram/blob/master/src/avram/charms/bool_extensions.cr) and [uuid](https://github.com/luckyframework/avram/blob/master/src/avram/charms/uuid_extensions.cr). I wonder if these are bugs that just haven't been caught yet 🤔

Reply:
I think they just use the default `from_db!` coming from `Avram::Type` https://github.com/luckyframework/avram/blob/master/src/avram/type.cr#L6

Reply:
ohh... sneaky 😂